plugins/playstation/src/playstation-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.PlayStation : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "playstation";
+	private const string MIME_TYPE = "application/x-cue";
```

Comment:
@Kekun good find! Right now the way we're getting the disc number is not very reliable... Will make a WIP PR so you guys can see my progress and discuss.